.github/workflows/android-ci.yml

```
@@ -0,0 +1,46 @@
+          echo '${{ secrets.LOCAL_PROPERTIES }}' >> ./local.properties
+
+#      - name: Generate google-services.json
+#        run: |
+#          echo '${{ secrets.GOOGLE_SERVICES }}' >> ./app/google-services.json
+
+#      - name: Code style checks
+#        run: |
+#          ./gradlew detekt
+
+      - name: Run build
+        run: ./gradlew buildDebug --stacktrace
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)